Library/DiscUtils.Vhdx/LogEntry.cs

```diff
@@ -121,7 +121,7 @@ public static bool TryRead(Stream logStream, out LogEntry entry)
             byte[] logEntryBuffer = new byte[header.EntryLength];
             Array.Copy(sectorBuffer, logEntryBuffer, LogSectorSize);
 
-            StreamUtilities.ReadFully(logStream, logEntryBuffer, LogSectorSize, logEntryBuffer.Length - LogSectorSize);
+            StreamUtilities.ReadExact(logStream, logEntryBuffer, LogSectorSize, logEntryBuffer.Length - LogSectorSize);
```

Comment:
Did not throw, does now. Bytes read is not tracked. Correct behavior is to throw on a short read to avoid treating zero-bytes as data.